Jint/Runtime/Interop/MethodDescriptor.cs

```diff
@@ -60,6 +60,17 @@ private static MethodDescriptor[] Prioritize(MethodDescriptor[] descriptors)
         {
             static int CreateComparison(MethodDescriptor d1, MethodDescriptor d2)
             {
+                // if its a generic method, put it on the end
+                if (d1.Method.IsGenericMethod)
```

Comment:
This is not stable and could potentially create infinite loops with some sort algorithm. The idea being that (a, b) might then be compared again with (b, a) and both would return opposite values if a == b.

If think the test should be that if d1 is generic and not d2, then put it at the end.